build/commands/scripts/listAffectedTests.js

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) 2025 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+const { getAffectedTests } = require('../lib/affectedTests')
+const config = require('../lib/config')
+
+module.exports = (program) =>
+  program
+    .command('list_affected_tests')
+    .description('prints the tests that need to be run given the file changes')
+    .option('-C [build_dir]', 'build config (out/Debug, out/Release)')
+    .option('--target_arch [target_arch]', 'target architecture')
```

Comment:
`target_arch` is not always sufficient to determine the build directory, you also might need target_os and target_environment. We're trying to get rid of this stuff, but for now it's needed cc @cdesouza-chromium

Reply:
I indeed forgot to add the other args. Added